### ISSUE TITLE
fast score editing not setting "draw", "win", "loss" - bug?

### ISSUE THREAD

**Issue by miroslawlis** [ThemeBoy/SportsPress#124]
When adding scores via wp-admin/edit.php?post_type=sp_event (fast editing).
"draw", "win" or "loss" are not set at all, (football aka soccer).
Can someone confirm that?

Cheers

**Comment by MounirHamani** [comment]
Yes I confirm that, In-Line editing doesn't transfer scores to the fields inside the post and when the fields inside the post are used to edit the score, the fields used for In-Line editing are reset to dashes.

**Comment by ThemeBoy** [comment]
@miroslawlis @MounirHamani you'll need to configure the outcomes by going to SportsPress > Configure then edit each outcome and choose a condition. For example, the condition for a win should be "Most Goals", etc.

**Comment by MounirHamani** [comment]
That condition is set, and it works properly you edit the score at ~~~~/post.php?post=105&action=edit but not when you do it through In-Line editing at ~~~~/edit.php?post_status=publish&post_type=sp_event .

**Comment by MounirHamani** [comment]
I tested on an other install and it works...

**Comment by MounirHamani** [comment]
Sorry for triple posting.

I went back near where you suggested, and I reset back the variable goals and then updated the other conditions and it was working again, so the problem came obviously from the variable goals because it didn't translate well.
Actually, the variable being invisible to the public and having control over the display title makes it enough for me not to pursue the bug further.

**Comment by miroslawlis** [comment]
@ThemeBoy got it.
Thank You.

**Comment by miroslawlis** [comment]
but... when You delete scores via fast editing (it will showing "-" again), "Outcome" will not be cleared, and will display last selected option (egz, Drow-drow)
